pkg/sdk/notification_integrations_dto_gen.go

```diff
@@ -0,0 +1,122 @@
+package sdk
+
+//go:generate go run ./dto-builder-generator/main.go
+
+var (
+	_ optionsProvider[CreateNotificationIntegrationOptions]   = new(CreateNotificationIntegrationRequest)
+	_ optionsProvider[AlterNotificationIntegrationOptions]    = new(AlterNotificationIntegrationRequest)
+	_ optionsProvider[DropNotificationIntegrationOptions]     = new(DropNotificationIntegrationRequest)
+	_ optionsProvider[ShowNotificationIntegrationOptions]     = new(ShowNotificationIntegrationRequest)
+	_ optionsProvider[DescribeNotificationIntegrationOptions] = new(DescribeNotificationIntegrationRequest)
+)
+
+type CreateNotificationIntegrationRequest struct {
+	OrReplace                *bool
+	IfNotExists              *bool
+	name                     AccountObjectIdentifier // required
+	Enabled                  bool                    // required
+	AutomatedDataLoadsParams *AutomatedDataLoadsParamsRequest
+	PushNotificationParams   *PushNotificationParamsRequest
+	EmailParams              *EmailParamsRequest
+	Comment                  *string
+}
+
+func (r *CreateNotificationIntegrationRequest) GetName() AccountObjectIdentifier {
+	return r.name
+}
+
+type AutomatedDataLoadsParamsRequest struct {
+	GoogleAutomatedDataLoad *GoogleAutomatedDataLoadRequest
+	AzureAutomatedDataLoad  *AzureAutomatedDataLoadRequest
+}
+
+type GoogleAutomatedDataLoadRequest struct {
+	GcpPubsubSubscriptionName string // required
```

Comment:
"PubSub" instead of "Pubsub"?

Reply:
I would leave it as is because `GCP_PUBSUB_SUBSCRIPTION_NAME`